pkg/plugins/plugins.go

```diff
@@ -346,19 +346,32 @@ func (s *PluginScanner) loadPlugin(pluginJSONFilePath string) error {
 	defer reader.Close()
 
 	jsonParser := json.NewDecoder(reader)
-	pluginCommon := PluginBase{}
-	if err := jsonParser.Decode(&pluginCommon); err != nil {
+	plugin := PluginBase{}
```

Comment:
Why the renaming? `pluginCommon` made sense, since it's just the `PluginBase` part.

Reply:
In that case, why not `pluginBase`?

Reply:
@wbrowne Makes no big difference, I think the idea is that "base" refers to a base class, while here "common" just refers to common fields (which can be loaded at this initial stage).

The main thing is to signal that this isn't the entire plugin. I know I was confused reading this code, until I realized that the whole plugin gets loaded later, so the explicit naming helps.

Reply:
Yeah I understand the hierarchy - hard to capture that with just a var name. I'll put it back as it was then for now